R/simulate.r

```diff
@@ -32,3 +32,26 @@ simulate_postgres <- function() {
     class = c("PostgreSQLConnection", "DBIConnection")
   )
 }
+
+# odbc connections --------------------------------------------------------
+
+setClass(
```

Comment:
Sure, quick question, do want me to use a single use function called `simulate_mssql`, or use the same code pattern as the other connections?:
```
simulate_postgres <- function() {
  structure(
    list(),
    class = c("PostgreSQLConnection", "DBIConnection")
  )
}
```